MonoGame.Framework/Color.cs

```diff
@@ -1732,19 +1732,19 @@ public static Color Multiply(Color value, float scale)
             return new Color((int)(value.R * scale), (int)(value.G * scale), (int)(value.B * scale), (int)(value.A * scale));
         }
 
-    /// <summary>
-    /// Gets a three-component <see cref="Vector3"/> representation for this object.
-    /// </summary>
-    /// <returns>A three-component <see cref="Vector3"/> representation for this object.</returns>
+        /// <summary>
+        /// Gets a <see cref="Vector3"/> representation for this object.
+        /// </summary>
+        /// <returns>A <see cref="Vector3"/> representation for this object.</returns>

```

Comment:
This one has the generic and unhelpful "for this object".  Instead we should give them useful information here:
```
/// Returns the color in a 0-to-1 normalized <see cref="Vector3"/> format.
```
Is a much more useful thing to say here.

I guess my greater point is we shouldn't waste words...  extra words take time for us to write and for developers to read...  they all should be useful words.